python/GafferUI/RampPlugValueWidget.py

```diff
@@ -52,8 +52,16 @@ def __init__( self, plug, **kw ) :
 		with column :
 			with GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Horizontal, spacing = 4 ) :
 				GafferUI.Spacer( IECore.V2i( 0 ), parenting = { "expand" : True } )
-				GafferUI.LabelPlugValueWidget( plug["basis"], horizontalAlignment=GafferUI.Label.HorizontalAlignment.Right )
-				GafferUI.SplineBasisPlugValueWidget( plug )
+
+				# TODO: Since we don't have a good way to register metadata on child plugs, we just write the
+				# metadata on this child plug right before constructing a widget for it.  There should probably
+				# be some way to do this genericly during initialization
+				Gaffer.Metadata.registerValue( plug['interpolation'],
```

Comment:
Can we pass `persistent = False` here so we don't pollute the serialisations with the metadata? I think that should make things easier to tidy up in the future if we get the registration mechanism we really need.